README.md

```diff
@@ -63,9 +70,7 @@ be good to go.
 Building the front-end
 ----------------------
 
-Run `grunt build` to regenerate the front-end logic and css files. For production,
-run with `grunt build:prod` to get minified versions, and run `grunt build:prod:true` to enable
-auto building whenever a front-end file changes. See `Gruntfile.js` for details.
+Run `npm postinstall` to download and regenerate the front-end. See `Gruntfile.js` for details.
```

Comment:
This is wrong. The `postinstall` hook means running `npm install` will kick off that script. They could also use `grunt requirejs`